server/CGameHandler.cpp

```diff
@@ -3995,8 +3995,8 @@ void CGameHandler::spawnWanderingMonsters(CreatureID creatureID)
 {
 	std::vector<int3>::iterator tile;
 	std::vector<int3> tiles;
-	gameState().getFreeTiles(tiles);
-	ui32 amount = tiles.size() / 200; //Chance is 0.5% for each tile
+	gameState().getFreeTiles(tiles, true);
+	ui32 amount = tiles.size() / 100; //Chance is 1% for each tile. Higher than the original 0.5% because there are fewer with nearby unguarded
```

Comment:
i do not think original game values should be changed

Reply:
True, original game values should be changed by  configurable settings instead.